### ISSUE TITLE
reproduce the accuracy in Pascal VOC

### ISSUE THREAD

**Issue by nlkim0817** [balancap/SSD-Tensorflow#97]
I have tried to reproduce the accuracy of the original caffe implementation in Pascal VOC 2007.
 1. training from the scratch
 2. training from the conv3 ~ + fine-tuning
 3. training from the conv6 ~ + fine-tuning
However, the accuracy was under 60 mAP.  (300x300 models)

Is anyone to success getting original performance?

**Comment by huangh12** [comment]
@nlkim0817  hello, have you found the problem now?

**Comment by nlkim0817** [comment]
@huangh12  I found some problems compared to original SSD implemented in CAFFE.
Now, I can get a similar result( - 3% accuracy to the original SSD ).

**Comment by huangh12** [comment]
@nlkim0817  I am still struggling with that.  Could do please point out where the difference from the SSD-caffe is?  ^_^

**Comment by heryms** [comment]
@nlkim0817 can you point out the problem？ thank you very much
